.github/workflows/lighthouse.yml

```diff
@@ -14,6 +14,7 @@ jobs:
     - uses: actions/checkout@v2
     - name: Build PWA distribution for production
       run: |
+          node -v
```

Comment:
Remove this?

Reply:
I added it because I could not find another easy way to check what node version the CI flow is running against.

Reply:
If you want to lock the node version, there's a GitHub action for it. I would use that, rather than echoing `node -v`, which will be buried in logs.

https://github.com/actions/setup-node

Reply:
The goal was not to lock the node version. Just to provide information about the current VM execution environment in case there are build errors.